network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1493,7 +1493,8 @@ def categories_page(self, request, slug):
         context['category'] = slug
         context['current_category'] = category
         context['products'] = products
-        context['pageTitle'] = f'{category} | ' + gettext("Privacy & security guide") + ' | Mozilla Foundation'
+        context['pageTitle'] = f'{gettext(category.name)} | ' + gettext("Privacy & security guide") + \
+                               ' | Mozilla Foundation'
```

Comment:
Hm, curious that we're using `f` strings as well as string concatenation here... let's also change this to just being a pure `f` string:
```suggestion
        context['pageTitle'] = f'{gettext(category.name)} | {gettext("Privacy & security guide")} | Mozilla Foundation'
```